samples/docker_aks/Makefile

```diff
@@ -5,10 +5,8 @@ APPDIR        := appdir
 BUILD_DIR     := $(CURDIR)
 CFLAGS        := -fPIC
 DOCKER_IMG    := mystikos-hello
-MYST_TARBALL := mystikos.tar.gz
-PSW_VERSION   := 2.15.100
```

Comment:
I've been trying to find where PSW_VERSION and MYST_RELEASE_VERSION was used in the Dockerfile but I don't see it anywhere. Were these just unused which is why you removed them?

Reply:
Yeah they were used in an older version